source/account_structure/index.html.md.erb

```diff
@@ -0,0 +1,80 @@
+---
+title: Account structure 
+weight: 55
+---
+
+# Account structure 
+
+If you have multiple services integrated, you can treat them separately or as
+a combined single service within GOV.UK and your Payment Service Provider
+(PSP). This gives 3 possible account configurations.
+
+As an example, take 2 services called "parking permits" and "parking fines": 
+
+### Type 1 (GOV.UK Pay separate, PSP separate)
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/accountstructure_1.png)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+both GOV.UK Pay and the PSP. 
+
+This is the default configuration for services. 
+
+In this setup, payments to services can go to separate bank accounts because
+each service has a PSP account. 
+
+### Type 2 (GOV.UK Pay combined, PSP combined)
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/accountstructure_2.png)
+
+"Parking permits" and "Parking fines" are considered as a single combined
+service in GOV.UK Pay and in the PSP. 
+
+You may want to use this configuration if you want to simplify the
+administration of multiple services in GOV.UK Pay, and have them appear as a
```

Comment:
"simplify the administration of multiple services in GOV.UK Pay" - yep, this describes it well. basically, they'll get all their transactions in one place - in the same transactions list and in the same CSV. they might need to configure their reference numbers differently eg parkingfine1234 and parkingpermit1234 if they need to separate out payments. 

"appear as a single item.." - i don't think this is really true. it'll appear as a single item regardless of the service setup